CONTRIBUTING.md

```diff
@@ -2,6 +2,19 @@
 
 ## Coding Style
 The citrine-python library follows [PEP8](https://www.python.org/dev/peps/pep-0008/), with the following exceptions:
-* Maximum line length is 99 characters
+* Maximum line length is 99 characters for code and 119 characters for docstrings
```

Comment:
Is this also intended for files in the `docs` folder? I find it's easiest to organize my thoughts when I use 1 sentence/line. It also makes it easier to edit later. However, this sometimes results in lines that exceed 119 characters.

Reply:
The docs/ folder is not checked by the linter, so this is not a strict requirement. But I do find it helpful to break up long sentences into multiple lines (to me, the important part is not having more than 1 sentence per line)

Reply:
👍 
Thanks for the clarification. That was not clear to me.